tests/integration/targets/nxos_static_routes/tests/common/deleted.yml

```diff
@@ -46,36 +46,5 @@
           - "'no ip route 192.0.2.64/28 192.0.2.23 name merged_route 1' in result.commands"
           - "result.commands | length == 3"
 
-    - ansible.builtin.include_tasks: _remove_config.yaml
-
-    - ansible.builtin.include_tasks: _populate_config.yaml
-
-    - name: Deleted (all routes)
```

Comment:
Could you please explain why we are deleting this test case?

Reply:
@NilashishC Thanks for your review, on an ideal device 
```
switch# show running-config | include '^ip(v6)* route'
switch# show running-config | section '^vrf context'
vrf context management
  ip name-server 192.168.255.1
  ip route 0.0.0.0/0 192.168.255.1
switch# 
```
there is  a static route entry as above under vrf management ^ Determining the IP address is challenging
And delete all would try to remove this entry and which pushes the appliance to an inaccessible state.

Reply:
Oh, you're right. We should not test "delete all" scenarios in integration tests for static routes.